feedback/readme.md

```diff
@@ -0,0 +1,143 @@
+- [] Apply techniques for receiving tough feedback
+
+## Set up 🌼
+
+- Pair up into groups of 2
+
+## The Activity 🏆 
+
+First, read [this article](https://tanzu.vmware.com/content/blog/mastering-feedback-in-the-workplace) on mastering feedback in the workplace. 
+
+Read the following scenarios and role play each. One person will be the person giving the feedback and one person will be receiving the feedback. After each scenario switch who is giving the feedback and who is receiving it.
+
+For the person giving feedback, remember to:
+- Be direct
+- Use clear language
+- Talk about behavior not personality
+- Be kind
+
+For the person receiving feedback, remember to:
+- Be available and approachaable
+- Accountable
+- Steer clear of defensiveness. 
+- Take a moment to reflect before respondng
+- Assume positive intent
+
+### Practice Giving Feedback
+
+1. Code Review
+
+Scenario: A team member submits the following code that is functional but has inconsistent formatting.
+
+```javascript
+function calculateTotal(price, tax, discount){
+    var t = price + tax
+    t = t - discount
+    console.log("Total: $" + t);
+    return t
+}
+
+let price = 100, tax = 5; let discount = 10; calculateTotal(price,tax,discount);
+```
+
+Objective: Practice giving constructive feedback on code quality, style consistency, and optimization, while maintaining a positive tone. Mention specific areas for improvement such as consistent use of semicolors, better variable declaration, and reduce the use of magic numbers. Acknowledge what was done well. 
+
+2. Missed Deadline
+
+Scenario: A teammate consistently misses deadlines and when they finally do deliver their tasks, they are often not of good quality tasks requiring the rest of the team to redo portions of their work.
+
+Objective: Try to use the situation, behavior, impact (SBI) method. Start by describing the situation, detailing the specific behavior observed, and explain the impact it had. Express concern in a non-blaming way, offer support, and work with them to identify potential roadblocks or strategies to improve. Practice addressing issues of accountability, responsibility, and time management without demotivating the person.
+
+3. Team Presentation
+
+Scenario: A team member gives a presentation that is informative but is too long-winded, going over an hour causing the audience to lose focus. This presentation will be used at the next client meeting that is only 1 hour long and needs to cover lots of other topics. 
+
+Objective: Try using the Observation, Feelings, Needs and Actions (OFNA) framework to give the team member feedback. Start by discussing youor observation, how it made you feel, what the needs are and the action they (or you together) can take to correct, Remember to only talk about your own feelings and not bring in others. Practice providing feedback on communication skills, structuring information, and presentation techniques.
```

Comment:
Let's structure this so each framework is introduced clearly

### Framework OFNA
1. Define the framework and link to an external resource
2. Give a worked example of using the framework
3. Then an exercise for the learner to apply the framework to
4. Then a quiz or check for understanding so the learner can assess their work

https://docs.google.com/document/d/18LF9yn-3Hge-96ITRAQJuPZiJ71tAOSDyO3MRyBJdIc/edit?usp=sharing